game.rb

```diff
@@ -0,0 +1,100 @@
+# game class
+class Game
+  
+  attr_accessor :column, :main_diagonal, :second_diagonal, :column_matrix
+
+  WIN = 'win'
+  DRAW = 'draw'
+  ON_GAME = 'on_game'
+
+  def initialize
+    @columns = []
+    @main_diagonal = []
+    @second_diagonal = []
+  end
+
+  def status(board)
+    
+    for i in board.table
+      if i.all?{ |inner| inner == 'X' }
+        return WIN
+      elsif i.all? { |inner| inner == 'O' }
+        return WIN
+      end
+    end
```

Comment:
[There are easier ways to check if all the elements of the array are equal](https://stackoverflow.com/questions/27471254/ruby-check-if-all-array-elements-are-equal).
You just need to check if all the elements are equal, not if they are equal to a particular symbol

Reply:
Actually, because of this check if you use a custom symbol the program won't work